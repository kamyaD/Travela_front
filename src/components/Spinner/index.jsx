import React from 'react';
import './index.scss';

const Spinner = () => (
  <div className="spiner--wrapper">
    <div className="spinner--large" />
  </div>
);

export default Spinner;
