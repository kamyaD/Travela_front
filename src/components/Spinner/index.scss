.spiner--wrapper {
  width: 100%;
  display: flex;
  justify-content: center;
}

.spinner--large {
  border: 2px solid #f3f3f3;
  border-top: 2px solid #385BD3;
  border-radius: 50%;
  width: 60px;
  height: 60px;
  animation: spin 2s linear infinite;
  margin-top: 40px;
}
