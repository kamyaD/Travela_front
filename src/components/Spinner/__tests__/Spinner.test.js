import React from 'react'
import Spinner from '..';


describe('<FilterContext />', () => {
  const wrapper = shallow(<Spinner />);

  it('should render without crushing', () => {
    const wrapper = shallow(<Spinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
