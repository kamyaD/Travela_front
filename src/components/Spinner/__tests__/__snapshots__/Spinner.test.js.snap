// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterContext /> should render without crushing 1`] = `
<div
  className="spiner--wrapper"
>
  <div
    className="spinner--large"
  />
</div>
`;
